hilti: uncaught exception, AssertionError with argument 'Test False' (from /home/robin/work/hilti/tests/.tmp/hilti.debug.assert-debug/assert-debug.hlt:11-11)
